src/test/java/com/znsio/teswiz/businessLayer/vodqa/VodqaBL.java

```diff
@@ -59,5 +59,11 @@ public VodqaBL verifyUserMoveToNextPage(String pageHeading) {
         assertThat(VodqaScreen.get().isPreviousPageHeadingVisible(pageHeading)).as(String.format("User is still on %s page", pageHeading)).isFalse();
         return this;
     }
+
+    public VodqaBL appWorksInBackground(int time) {
+        LOGGER.info("Validating wheter app works in background or not");
+        VodqaScreen.get().validateAppWorkInBackground(time);
```

Comment:
validation is not done, whether app was working in background or not.
Add assertions and validate the functionality

Reply:
done